docs/system-admin-guide/custom-fields/README.md

```diff
@@ -84,12 +87,58 @@ To **delete** a custom field, click on the delete icon next to the respective cu
 
 ## Create a multi-select custom field
 
-For work package custom fields of type **List** and **User** you may also select **multi-select custom fields** to select more than one value at once.
+For work package custom fields of type **List**, **User**, **Version** and **Hierarchy** you may also select **multi-select custom fields** to select more than one value at once.
 
-To create a multi-select custom field follow the same steps as you would when [creating a standard custom field](#add-a-new-custom-field). Select format **List, User or Version** and check the option *Allow multi-select*.
+To create a multi-select custom field follow the same steps as you would when [creating a standard custom field](#add-a-new-custom-field). Select format **List**, **User**, **Version** or **Hierarchy **and check the option *Allow multi-select*.
 
-![Sys-admin-multi-select-custom-field](system-admin-allow-multi-select.png)
+![Allow multi-select values for a custom field in OpenProject](system-admin-allow-multi-select.png)
 
 When using multi-select custom fields, you can add as many options as required. The cross icon next to an option will remove it from the selection. The check mark will save your changes.
 
 ![Multi select custom field in a work package](system-guide-custom-field-work-package.png)
+
+## Create a Hierarchy custom field (Enterprise add-on)
```

Comment:
🟡 Let's not optimize this now, but is it weird to read, that explaining the Hierarchy is on the same headline level like explaining "Edit or remove a custom field" and "Configure my custom field to be multi-select"?

I'd rather expect one header containing all porperties below (regexes, mutli-select, filter- and searchable, etc.), and another one explaining the formats?